v1.1/admin_ui_replication_dashboard.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Replication Dashboard
+toc: false
+feedback: false
+---
+
+CockroachDB Admin UI enables you to monitor the replication metrics for your cluster.
```

Comment:
Same comment as for Cluster Overview dashboard: We ca combine the text before and after the table of contents:

> The **Replication** dashboard in the Admin UI lets you metrics about how data is replicated in your cluster. To view this dashboard, [access the Admin UI](explore-the-admin-ui.html#access-the-admin-ui) and then select **Dashboard > Replication**.

Can you make the same type of change to the other dashboard pages?